core/src/Pos/Core/Common/StakeholderId.hs

```diff
@@ -1,10 +1,27 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
@erikd @mhuesch I'm a bit confused as to why these instances are still considered orphans. Is it because `StakeholderId` is a type alias for `AddressHash PublicKey`, meaning that these instances would actually have to be defined alongside `AddressHash`?

Reply:
Right - both `AddressHash` and `PublicKey` are defined in other modules, and `StakeholderId` is just a type alias for them.

We encountered something similar before with `DataMsg`, and I think the solution was the put the instances with the type `a` for which we were defining `instances Foo (DataMsg a) where`.